content/docs/introducing-jsx.md

```diff
@@ -6,31 +6,37 @@ prev: hello-world.html
 next: rendering-elements.html
 ---
 
-Consider this variable declaration:
+Beschouw de volgende toewijzing van een variabele:
```

Comment:
Personally I think "toewijzing" should be one of "verklaring", "declaratie" here since there's a bit of a semantic difference.

Reply:
Then I prefer `declaratie`. Will use that.